src/main/java/uk/ac/ebi/subs/biostudies/client/BioStudiesSession.java

```diff
@@ -2,88 +2,112 @@
-            );
-        } catch (HttpServerErrorException e) {
-            logHttpError(e, "Http server error during createupdate");
-            throw e;
-        } catch (HttpClientErrorException e) {
-            logHttpError(e, "Http client error during createupdate");
-            throw e;
+                getRequestUri(dataOwner),
+                createBioStudiesSubmitRequest(bioStudiesLoginResponse.getSessid(), bioStudiesSubmission),
+                BioStudiesSubmission.class);
+        } catch (HttpServerErrorException | HttpClientErrorException httpError) {
+            logHttpError(httpError, "Http server error during create/update");
+            throw httpError;
         }
 
         logSubmissionResponse(response);
 
         return response.getBody();
     }
 
-    public SubmissionReport update(BioStudiesSubmission submission) {
-        BioStudiesSubmissionWrapper wrapper = new BioStudiesSubmissionWrapper();
-        wrapper.getSubmissions().add(submission);
+    public BioStudiesSubmission update(BioStudiesSubmission submission) {
         return restTemplate.postForObject(
-                bioStudiesConfig.getServer() + "/submit/createupdate" + "?BIOSTDSESS=" + bioStudiesLoginResponse
-                        .getSessid(),
-                wrapper,
-                SubmissionReport.class
-        );
+            bioStudiesConfig.getServer() + "/submissions",
+            createBioStudiesSubmitRequest(bioStudiesLoginResponse.getSessid(), submission),
+            BioStudiesSubmission.class);
     }
 
     public BioStudiesSubmission getSubmission(String accNo) {
-        return restTemplate.getForObject(
-                bioStudiesConfig.getServer() + "/submission/" + accNo + "?BIOSTDSESS=" + bioStudiesLoginResponse
-                        .getSessid(),
-                BioStudiesSubmission.class);
+        return restTemplate.exchange(
+            bioStudiesConfig.getServer() + "/submissions/" + accNo + ".json",
+            HttpMethod.GET,
+            new HttpEntity(getSecuredHeaders(bioStudiesLoginResponse.getSessid())),
+            BioStudiesSubmission.class).getBody();
+    }
+
+    private URI getRequestUri(DataOwner dataOwner) {
+        Map<String, String> parameters = new LinkedHashMap<>();
+        parameters.put("sse", "true"); //enables super user actions
+        parameters.put("onBehalf", dataOwner.getEmail());
+        parameters.put("name", dataOwner.getName());
+
+        List<String> params = parameters.entrySet().stream()
+            .map(entry -> entry.getKey() + "=" + UriUtils.encodeQueryParam(entry.getValue(), "UTF-8"))
+            .collect(Collectors.toList());
+
+        String queryString = "?" + String.join("&", params);
+
+        return URI.create(bioStudiesConfig.getServer() + "/submissions" + queryString);
+    }
+
+    private HttpEntity<String> createBioStudiesSubmitRequest(String sessionId, BioStudiesSubmission submission) {
+        ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
object mapper is a super heavy instance but it is thread-safe so we can set as a field parameter and reuse here. (To do not created it in each request)